### PR TITLE
Certificates - CovPass Information Adjustments (EXPOSUREAPP-9430)

### PR DESCRIPTION
**Pull request by BMItr**
Feature Branch for EXPOSUREAPP-9430, containing all subtasks (do not merge yet)

https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-9430

Screenshot: (additional Info on top of the QR Code)

image

some smaller style adjustments regarding certificate views: (top text & name)

image

adding the name of the cert owner (Cert Details View):

image

### CODE REVIEW

**Review by AlexanderAlferov — DISMISSED** [review]
🚀

**Review by jurajkusnier — COMMENTED** [review]
Looks good, just a few questions. And also small part of the text is not visible on my device 
image

File: Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/person/ui/details/PersonDetailsFragment.kt [hunk 1]
```diff
@@ -115,21 +117,22 @@ class PersonDetailsFragment : Fragment(R.layout.person_details_fragment), AutoIn
...
         val deviceWidth = requireContext().resources.displayMetrics.widthPixels
 
         val layoutParamsRecyclerView: CoordinatorLayout.LayoutParams = binding.recyclerViewCertificatesList.layoutParams
             as (CoordinatorLayout.LayoutParams)
 
-        val textParams = binding.title.layoutParams as (LinearLayout.LayoutParams)
+        val textParams = binding.toolbarLinearLayout.layoutParams as (CollapsingToolbarLayout.LayoutParams)
 
         val divider = 2
         textParams.bottomMargin = (deviceWidth / divider) - 24 /* 24 is space between screen border and Card */
```
[the remark this answers is not in the document]
Comment (jurajkusnier) [reply Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/person/ui/details/PersonDetailsFragment.kt:134]: actually, this is not your change, never mind

File: Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/covpass/CovPassInfoFragment.kt [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+        super.onViewCreated(view, savedInstanceState)
+
+        binding.apply {
+
+            toolbar.setNavigationOnClickListener { popBackStack() }
+
+            linkToFaq.setTextWithUrl(
+                R.string.cov_pass_info_faq_link,
+                R.string.cov_pass_info_faq_link,
+                R.string.vaccination_card_booster_eligible_faq_link // TBD
```
[the remark this answers is not in the document]
Comment (BMItr) [reply Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/covpass/CovPassInfoFragment.kt]: yes its final.. was discussed but axel probably forgot to remove the tbd. removed

**Review by jurajkusnier — APPROVED** [review]

**Review by mtwalli — APPROVED** [review]

**Comment by MikeMcC399** [comment]
@mtwalli 
You probably already know that the text display problems mentioned in https://github.com/corona-warn-app/cwa-app-android/pull/4087#pullrequestreview-760945532 by @jurajkusnier are not yet fixed. But just in case this has slipped by ...

The problem is still reproducible in release/2.11.x with the latest commit (https://github.com/corona-warn-app/cwa-app-android/commit/8eeb2f79b3f6c7cf7c92c7d370480041c02b1034).
Run `PersonOverviewFragmentTest` on Google Pixel 3a emulator with Android 11 and font size and display size both set to Default.

PersonOverviewFragment_many_persons

Setting font size to largest and display size to larger gives a more extreme view where much more of the text is cut off.

image

**Comment by BMItr** [comment]
This Problem has already been reported (Exposureapp-9628).
Will be fixed by upcoming PR (branch: fix/9628-overlapping)

**Comment by MikeMcC399** [comment]
@BMItr 
Thanks for the pointer to [fix/9628-overlapping](https://github.com/corona-warn-app/cwa-app-android/tree/fix/9628-overlapping). That seems to fix it fine! 👍🏻

**Pull request merged**